<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>用户昵称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_username"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>号码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_peername"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_call">
        <property name="text">
         <string>拨号</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="OpenGLWidget" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_exit"/>
    <addaction name="action_log"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="action_font"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_about"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_serverSet"/>
    <addaction name="separator"/>
    <addaction name="action_userInfoSet"/>
    <addaction name="action_collectionSet"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_serverSet">
   <property name="text">
    <string>服务器设置</string>
   </property>
  </action>
  <action name="action_font">
   <property name="text">
    <string>字体设置</string>
   </property>
  </action>
  <action name="action_userInfoSet">
   <property name="text">
    <string>用户信息设置</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>查看帮助</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_collectionSet">
   <property name="text">
    <string>采集设备设置</string>
   </property>
  </action>
  <action name="action_log">
   <property name="text">
    <string>日志</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>openglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
